tests/tests_custom_certificate.yaml

```diff
@@ -0,0 +1,50 @@
+---
+- name: Generate certificate using certificate role
```

Comment:
this doesn't have to be a separate play - this can be a task in the `Test PostgreSQL server with ssl support using certificate role` play

There is a way to use the `certificate` role in test mode which makes it easier to use for system roles testing.

```yaml
        - name: Generate certificates
          include_role:
            name: fedora.linux_system_roles.certificate
          vars:
            certificate_requests:
              - name: /tmp/mycert
                dns: www.example.com
                ca: self-sign
            certificate_test_mode: true
```

then you don't need the task for `Stop tracking certificate`

Reply:
Thanks, I didn't know it.